src/integrationtests/java/com/aws/greengrass/integrationtests/lifecyclemanager/GenericExternalServiceIntegTest.java

```diff
@@ -58,7 +68,7 @@
 import static org.mockito.Mockito.verify;
 
 
-class GenericExternalServiceTest extends BaseITCase {
+class GenericExternalServiceIntegTest extends BaseITCase {
```

Comment:
I renamed this since the unit test class is also named `GenericExternalServiceTest` and it's a little annoying to run individual tests